extensions/cdc-postgres/pom.xml

```diff
@@ -82,6 +82,16 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.postgresql</groupId>
+                <artifactId>postgresql</artifactId>
+                <version>${postgresql.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
I've moved it for all modules but:
1. Spring - it's pom import not just version enforce, so IMHO it's quite different
2. Elastic 5 - because each Elastic version needs other version of lucene queryparser